.drone.jsonnet

```diff
@@ -488,29 +488,12 @@ local conformance_pipelines = [
   Pipeline('cron-conformance-qemu', default_pipeline_steps + [conformance_k8s_qemu], [default_cron_pipeline]) + cron_trigger(['nightly']),
 ];
 
-// Cloud images pipeline.
-
-local cloud_images = Step("cloud-images", depends_on=[load_artifacts], environment=creds_env_vars);
-
-local upload_images_steps = default_pipeline_steps + [
-  cloud_images,
-];
-
-
-local upload_images_trigger = {
-  trigger: {
-    target: {
-      include: ['upload-images'],
-    },
-  },
-};
-
-local upload_images_pipeline = Pipeline('upload-images', upload_images_steps) + upload_images_trigger;
-
 // Release pipeline.
 
 local boot = Step('boot', depends_on=[e2e_docker, e2e_qemu]);
 
+local cloud_images = Step("cloud-images", depends_on=[load_artifacts], environment=creds_env_vars);
```

Comment:
I think `depends_on` should be updated with same dependencies as steps around, `e2e_docker, e2e_qemu`

Reply:
yep, good call. fixed in amended commit.